demo/KevinDemo/Stories/AccountLinking/AccountLinkingView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  AccountLinkingView.swift
+//  KevinDemo
+//
+//  Created by Daniel Klinge on 10/06/2022.
+//
+
+import SwiftUI
+
+public struct AccountLinkingView: View {
+    
+    @StateObject var viewModel = AccountLinkingViewModel()
+    
+    public var body: some View {
+        NavigationView {
+            ZStack {
+                Color("PrimaryBackgroundColor").ignoresSafeArea()
+                    .zIndex(-1)
+
+                if viewModel.viewState.isLoading {
+                    ProgressView()
+                } else if let linkedBanks = viewModel.viewState.linkedBanks, linkedBanks.isEmpty == false {
+                    AccountsView(
+                        linkedBanks: linkedBanks.toArray(),
+                        linkBank: viewModel.invokeAccountLinkingSession,
+                        deleteBank: viewModel.deleteLinkedBank
+                    )
+                    .navigationBarTitle("kevin_window_link_account_title".localized(), displayMode: .large)
+                } else {
+                    NoAccountsView(onLinkAccountAction: {
+                        viewModel.invokeAccountLinkingSession()
+                    })
+                }
+            }
+        }
+        .animation(.easeInOut(duration: 0.15))
+        .sheet(isPresented: $viewModel.viewState.openKevin, content: {
+            if let controller = viewModel.kevinController {
+                KevinViewControllerRepresentable(controller: controller)
+                    .presentation(canDismissSheet: false)
+            }
+        })
+        .alert(isPresented: $viewModel.viewState.showMessage, content: {
+            Alert(
```

Comment:
this is deprecated use `alert(_:isPresented:presenting:actions:message:)` instead

Reply:
`alert(_:isPresented:presenting:actions:message:)` is only available for iOS 15+, but demo app supports iOS 14